data/tutorials/platform/bp_03_run_executables_and_tests.md

```diff
@@ -43,7 +43,7 @@ You can run it with `dune exec bin/main.exe` or `dune exec my-app`.
 
 The `dune exec <executable_path>.exe` can take some time to compile a project when multiple files are involved. It can be beneficial to have a process that recompiles files as soon as they are changed. Then after the last saved file, it is possible that it remains only a last file to compile or just the linking process of all files and libraries.
 
-The command `dune build --watch` looks for things that need to be compiled, and afterwards, just waits for a modified file that will trigger the compilation of needed modules.
+The command `dune build --watch` overwatches for things that need to be compiled. It waits for a file modification that triggers the compilation of needed modules.
```

Comment:
I don't think overwatches is word used outside of military tactics. Oversees things? Or just "...watches for things"?